src/test/java/seedu/address/logic/commands/VolunteerDeleteCommandTest.java

```diff
@@ -7,8 +7,8 @@
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
 import static seedu.address.logic.commands.CommandTestUtil.showVolunteerAtIndex;
 import static seedu.address.testutil.TypicalEvents.getTypicalEventStorage;
-import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_VOLUNTEER;
-import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_VOLUNTEER;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_VOLUNTEER_OR_EVENT;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_VOLUNTEER_OR_EVENT;
```

Comment:
max is using `INDEX_FIRST_EVENT` and `INDEX_SECOND_EVENT` etc. need to standardise changes

Reply:
so i talked to max about this and my suggestion is to change these to `INDEX_FIRST` etc. see if yall agree

Reply:
will update accordingly